src/mocks.ts

```diff
@@ -29,6 +30,23 @@ export const mocks = [];`);
   return mocks as IMock[];
 }
 
+export function getMockForJSON(json: string | {}): RequestHandler {
+  if (!json) {
+    throw new Error("JSON argument is missing!");
+  }
+
+  let jsonString = json;
+
+  if (typeof json !== "string") {
```

Comment:
If I get this correctly, `getMockForJSON("foo")` would result in an invalid output. so if a string is passed, you should try something like `JSON.stringify(JSON.parse(json))`.